platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -60,6 +60,18 @@ We don't expect too many occurrences given the similar nullability semantics.
 Note that plugins using JSR-305 annotations on their types will continue to work as they did with previous Gradle versions.
 In other words, it is still supported to use `javax.annotations` for nullability on your plugin types such as tasks or extensions.
 
+==== Gradle no longer implicitly builds certain artifacts when running assemble
+In previous versions of Gradle, the `assemble` task would implicitly build artifacts of any configuration where the `visible` flag was not set to `false`.  This behavior has been removed in Gradle 9.0.  If you have a custom configuration whose artifacts should be built when running `assemble`, you should add a dependency from the artifact to the `assemble` task.
```

Comment:
> you should add a dependency from the artifact to the `assemble` task.

Can we add a short code snippet showing this off?

Reply:
Yep.  Done.